apidump/apidump.go

```diff
@@ -76,6 +76,61 @@ type Args struct {
 	Plugins []plugin.AkitaPlugin
 }
 
+func DumpPacketCounters(interfaces map[string]interfaceInfo, inboundSummary *trace.PacketCountSummary, outboundSummary *trace.PacketCountSummary) {
+	// Using a map gives inconsistent order when iterating (even on the same run!)
+	directions := []kgxapi.NetworkDirection{kgxapi.Inbound, kgxapi.Outbound}
+	toReport := []*trace.PacketCountSummary{inboundSummary}
+	if outboundSummary != nil {
+		toReport = append(toReport, outboundSummary)
+	}
+
+	// Debugging info: packets per interface and prot
+	printer.Stderr.Debugf("==================================\n")
```

Comment:
Make this span the whole width of the table? Similarly elsewhere.

Reply:
Fixed in https://github.com/akitasoftware/akita-cli/pull/18/commits/956c721b9bd0c25cc7d45c5583260cf4c5a70f3c

(It spans the width of the narrower table in its original form; I wanted all three the same, so now they're all longer.)